ci-operator/step-registry/openshift/release/analysis/test-case-analysis/openshift-release-analysis-test-case-analysis-commands.sh

```diff
@@ -179,10 +163,19 @@ job-run-aggregator analyze-test-case \
 	--job-start-time=${JOB_START_TIME} \
 	--working-dir=${ARTIFACT_DIR}/aws-proxy \
 	--timeout=4h30m \
-	--test-group=${TEST_GROUP}
-if [ $? -gt 0 ]; then
-	ret=$?
-fi
+	--test-group=${TEST_GROUP} &
+PIDS="$PIDS $!"
+
+echo "Waiting for pids to complete: $PIDS"
+ret=0
+for pid in $PIDS
+do
+  echo "Waiting for $pid"
```

Comment:
optional suggestion (because we might be curious how long the one we're waiting on took): ``echo "`date`: Waiting for $pid"`` and then right before the `done`, ``echo "`date`: $pid finished"``